{'exception_type': 'TypeError', 'exception_message': 'list indices must be integers or slices, not tuple', 'traceback': 'Traceback (most recent call last):\n  File "c:\\Users\\ezzak\\Desktop\\Study\\FISE A2\\Commande Entreprise\\repo\\Go-Game-Stream-Recognition-\\main.py", line 26, in processing_thread\n    game_plot = game.main_loop(ProcessFrame)\n  File "c:\\Users\\ezzak\\Desktop\\Study\\FISE A2\\Commande Entreprise\\repo\\Go-Game-Stream-Recognition-\\GoGame.py", line 36, in main_loop\n    self.process_frame(frame)\n  File "c:\\Users\\ezzak\\Desktop\\Study\\FISE A2\\Commande Entreprise\\repo\\Go-Game-Stream-Recognition-\\GoGame.py", line 61, in process_frame\n    input_points = get_corners(self.results)\n  File "c:\\Users\\ezzak\\Desktop\\Study\\FISE A2\\Commande Entreprise\\repo\\Go-Game-Stream-Recognition-\\processing.py", line 817, in get_corners\n    corner_boxes = are_corner_inside_box(corner_boxes_, np.array(board_model_edges))\n  File "c:\\Users\\ezzak\\Desktop\\Study\\FISE A2\\Commande Entreprise\\repo\\Go-Game-Stream-Recognition-\\processing.py", line 793, in are_corner_inside_box\n    square_x1 = corner_boxes[:, 0]\nTypeError: list indices must be integers or slices, not tuple\n'}